packages/discord.js/typings/index.d.ts

```diff
@@ -2454,7 +2452,7 @@ export class Webhook extends WebhookMixin() {
   public avatar: string;
   public avatarURL(options?: ImageURLOptions): string | null;
   public channelId: Snowflake;
-  public client: Client;
+  public readonly client: Client;
```

Comment:
This isn't a getter but a property defined through Object.defineProperty, should it be documented this way or with the Readonly type?

Reply:
Well, it isn't `writable` but it also isn't a getter so it should be like this. I think Readonly would only be used with `Object.freeze`

Reply:
That's correct, `Object.defineProperty` by default assigns readonly properties.